includes/class-admin.php

```diff
@@ -114,4 +128,4 @@ public function order_details_access( $fields ) {
 	}
 
 }
-$affiliatewp_menu = new AffiliateWP_Order_Details_For_Affiliates_Admin;
+$affiliatewp_menu = new AffiliateWP_Order_Details_For_Affiliates_Admin();
```

Comment:
```suggestion
$affiliatewp_menu = new AffiliateWP_Order_Details_For_Affiliates_Admin;
```
Local empty instantiation style.